<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xVSessionDlg</class>
 <widget class="QDialog" name="xVSessionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Session Management</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../xVTK_resources.qrc">:/images/vtk-logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Incomplete or stored sessions found. 
Do you like to restore data?</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="pSessionTreeWdgt">
     <property name="font">
      <font>
       <family>URW Gothic L</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="pClearAllIncompleteSessionsCB">
     <property name="text">
      <string>delete all incomplete sessions</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QToolButton" name="pDeleteTB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="pCancelTB">
       <property name="text">
        <string>&amp;cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pLoadTB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pSessionTreeWdgt</tabstop>
  <tabstop>pClearAllIncompleteSessionsCB</tabstop>
  <tabstop>pDeleteTB</tabstop>
  <tabstop>pCancelTB</tabstop>
  <tabstop>pLoadTB</tabstop>
 </tabstops>
 <resources>
  <include location="../xVTK_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pCancelTB</sender>
   <signal>clicked()</signal>
   <receiver>xVSessionDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
